jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/PropertiesUtil.java

```diff
@@ -39,4 +39,20 @@ public static Properties getPropertiesFromResource(URL resource) {
     }
     return ret;
   }
+
+  /**
+   * Return first Non Null set property from a set of provided properties
+   *
+   * @param properties {@link Properties} in a given project
+   * @param keys an array of property key values to find
+   * @return a string which is first non null value found for the provided list
+   */
+  public static String getValueFromProperties(Properties properties, String[] keys) {
```

Comment:
Changing to:
```suggestion
  public static String getValueFromProperties(Properties properties, String... keys) {
```
will make life easier for method users and invocations will look clearer.